src/pages/eth2/index.js

```diff
@@ -330,79 +333,84 @@ const ResearchContainer = styled.div`
 const paths = [
```

Comment:
One general thought / feedback here - I think it'd be less duplication if we just updated these keys (e.g. from `title` to `titleId`, then have the value be the id string vs including a `<Translation/>` component every time.

e.g. so instead of:
```
const paths = [
  {
    emoji: ":rocket:",
    title: <Translation id="page-eth2-scalable" />,
    description: <Translation id="page-eth2-scalable-desc" />,
  },
  {
    emoji: ":shield:",
    title: <Translation id="page-eth2-secure" />,
    description: <Translation id="page-eth2-secure-desc" />,
  },
  {
    emoji: ":evergreen_tree:",
    title: <Translation id="page-eth2-staking-sustainability" />,
    description: <Translation id="page-eth2-sustainable-desc" />,
  },
]
```
Something like:
```
const paths = [
  {
    emoji: ":rocket:",
    titleId: "page-eth2-scalable",
    descriptionId: "page-eth2-scalable-desc",
  },
  ...
]
```
THEN in the render, just passing the id to a Translation component, e.g.
```
<Translation id={path.titleId}/>
```
Make sense?

Reply:
Yes, I can do that going forward.